src/markdown/events/fof-figma-barcelona-apr-2021.mdx

```diff
@@ -0,0 +1,141 @@
+---
+draft: false #required (true or false)
+
+#Event Card Fields
+title: 'Friends of Figma Barcelona' #required
+featuredImage: '/images/events/figma-barcelona-banner.png' #required
```

Comment:
@drakonux I believe you changed this filename right? `static/images/events/fofbcn-banner.png`

just realized why that image wasnt showing up 😬